### DISCUSSION TITLE
Is there support of Transliteration in ICU4X?

### DISCUSSION THREAD

**Discussion by akshdeep-singh** [unicode-org/icu4x#3763] [Q&A] [answered]
Hi, is there support of [Transliteration](https://unicode-org.github.io/icu/userguide/transforms/general/) in ICU4X yet?

**Comment by Manishearth** [comment, marked as answer]
Not yet, we're working on it and should have a working experimental component in a month or so.

You can track progress in https://github.com/unicode-org/icu4x/issues/3736
